src/apolo_app_types/protocols/common/abc_.py

```diff
@@ -25,12 +23,13 @@ class AppInputsDeployer(BaseModel):
 
 class AppInputs(BaseModel):
     model_config = ConfigDict(arbitrary_types_allowed=True, protected_namespaces=())
-
-    def __init_subclass__(cls: type["AppInputs"], **kwargs: Unpack[ConfigDict]) -> None:
-        """Validate field types at class definition time."""
-        super().__init_subclass__()
-
-        validate_complex_type_prop(cls)
+    #
+    # def __init_subclass__(cls: type["AppInputs"],
```

Comment:
should we drop it?

Reply:
Just kept it out of curiosity, because I am not sure everything will work on BE